conversions.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto2";
+
+package cakeproto;
+
+import "common.proto";
+
+message Conversion {
+  optional int64 id = 1;
+  optional int64 visitor_id = 2;
+  optional int64 request_session_id = 3;
+  optional int64 click_id = 4;
+  optional int64 click_request_session_id = 5;
+  optional TimeWithZone conversion_date = 6;
+  optional TimeWithZone click_date = 7;
+  optional TimeWithZone last_updated = 8;
+
+  optional IdName event = 9;
+  optional IdName affiliate = 10;
+  optional IdName advertiser = 11;
+  optional IdName offer = 12;
+  optional IdName offer_contract = 13;
+  optional int64 campaign_id = 14;
+  optional IdName campaign_type = 15;
+  optional IdName creative = 16;
+
+  optional string sub_id1 = 17;
+  optional string sub_id2 = 18;
+  optional string sub_id3 = 19;
+  optional string sub_id4 = 20;
+  optional string sub_id5 = 21;
+
+  // click_ip_address and click_user_agent omitted - see associated Click object
```

Comment:
No, lite-clicks don't have a click object. Okay, they *could* be useful for the combination of lite clicks and s2s conversions (because the conversion IP and UA then is from the Adv-server, not the user).